sale_exception/data/sale_exception_data.xml

```diff
@@ -31,4 +31,22 @@
         >if obj.product_id and sale_line.product_id.type == 'product' and obj.product_id.with_context(warehouse=obj.order_id.warehouse_id.id).virtual_available &lt; obj.product_uom_qty: failed=True</field>
```

Comment:
The module doesn't depend on stock, but has this exception, so it's inconsistent

Reply:
Exactly, this exception is incorrect because it depends on stock. 
Maybe you should create a new addon called: `sale_exception_stock` with the dependency on `sale_stock` and move that exception there. What do you think?